File: TaskManager/ClientApp/app/components/shared/models/project.ts
export class Project {
	projectID: number;
	projectName: string;
	bankID: number;
	bankName: string;
	contactPersonID: number;
	contactPersonName: string;
	director: string;
	phoneFaxes: string;
	
	constructor() {
		this.projectID = 0;
		this.projectName = "";
		this.bankID = 0;
		this.bankName = "";
		this.contactPersonID = 0;
		this.contactPersonName = "";
		this.director = "";
		this.phoneFaxes = "";
	}
}

export interface IOneProject {
	project: Project;
}

File: TaskManager/Models/Project.cs
using System.Collections.Generic;

namespace TaskManager.Models
{
	public class Project
	{		
		public int ProjectID { get; set; }
		public string ProjectName { get; set; }
		public int? BankID { get; set; }
		public Bank Bank { get; set; }
		public int? ContactPersonID { get; set; }
		public Person ContactPerson { get; set; }
		public string Director { get; set; }
		public string PhoneFaxes { get; set; }
		public ICollection<Task> Tasks { get; set; } = new List<Task>();
		public ICollection<ProjectFile> ProjectFiles { get; set; } = new List<ProjectFile>();

		public string BankName
		{
			get
			{
				if (Bank != null)
					return Bank.BankName;
				else
					return "";
			}
		}

		public string ContactPersonName
		{
			get
			{
				if (ContactPerson != null)
					return ContactPerson.FIO;
				else
					return "";
			}
		}
	}
}
